### DISCUSSION TITLE
MX29L3211 Unknown Flashrom

### DISCUSSION THREAD

**Discussion by Khian** [sanni/cartreader#106] [Need help]
![IMG_7727](https://user-images.githubusercontent.com/6051116/112329974-f0b17e80-8cb7-11eb-990a-e982be7cc8a3.jpg)

I'm getting this error trying to flash an MX29L3211 both of SMD switches are on top position.

Any idea??? 
Thx!!

**Comment by sanni** [comment]
Did you do the modification needed on the Arduino Mega do allow for 3.3V and also installed the AMS 1117 3.3 on the Cart Reader PCB and switched the voltage to 3.3V on the voltage switch?
Also I can't tell if the CLK and EEP switches are soldered in and in the right mode.

**Reply by Khian** [reply]
Thanks Sanni for you reply! I think that I have everithing all right... I post more pics

Thx!

![IMG_7821](https://user-images.githubusercontent.com/6051116/112452793-4a1cba80-8d57-11eb-8aee-7d2d235bfec1.jpg)
![IMG_7822](https://user-images.githubusercontent.com/6051116/112452797-4ab55100-8d57-11eb-930e-11383ac76106.jpg)
![IMG_7823](https://user-images.githubusercontent.com/6051116/112452798-4b4de780-8d57-11eb-9ae7-ec1c62c3bf52.jpg)

**Comment by sanni** [comment]
You can try dumping a SNES HiROM game cartridge and read/write its SRAM since the Flashrom Programmer is based on that pinout to see if the problem is with the Cart Reader PCB or with the Flashrom Adapter.
